framework/src/modules/base_command.ts

```diff
@@ -17,6 +17,9 @@ import { Schema, emptySchema } from '@liskhq/lisk-codec';
 import { CommandVerifyContext, CommandExecuteContext, VerificationResult } from '../state_machine';
 import { NamedRegistry } from './named_registry';
 
+/**
+ * The `BaseCommand` is the class every module command extends from.
```

Comment:
A minor point; as `BaseCommand` enforces implementation of an interface, could some info about it be added/rephrased, something of sort; `The `BaseCommand` represents Lisk commands, that each module command extends from.`

Reply:
```suggestion
 * The `BaseCommand` represents Lisk commands by providing a generic interface, from which each command extends from.
```